ams/dashboard/src/views/catalogs/Detail.vue

```diff
@@ -265,6 +265,13 @@ const authConfigMap = {
   'auth.kerberos.krb5': 'Kerberos Krb5'
 }
 
+const defaultPropertiesMap = {
+  ams: 'warehouse',
```

Comment:
It's better to use a list as value to store these properties for each key (ams,hadoop...), in order to support more required properties for each key in the future.

Reply:
@wangtaohz  Thanks for your comment, I've updated the code, please review again.